pkg/sql/importer/import_processor.go

```diff
@@ -396,6 +396,14 @@ func ingestKvs(
 
 	writeTS := hlc.Timestamp{WallTime: spec.WalltimeNanos}
 
+	// TODO(jeffswenson): there's some bug between import and merge regarding filenames
+	// ERROR: open /Users/swenson/local/3/data/extern/import/1050679199897354241/customers rows0.sst: no such file or directory: nodelocal storage file does not exist: external_storage: file doesn't exist
+	// 	% find ./3/data/extern
+	// ./3/data/extern
+	// ./3/data/extern/import
+	// ./3/data/extern/import/1050679199897354241
+	// ./3/data/extern/import/1050679199897354241/0.sst
```

Comment:
This is caused by (e *ExternalFileAllocator) AddFile, the filename is computed as: fileName := fmt.Sprintf("%d.sst", fileIndex)...We need to append the last part of the path